airbyte-cdk/python/airbyte_cdk/sources/declarative/parsers/model_to_component_factory.py

```diff
@@ -515,7 +515,7 @@ def create_dpath_extractor(self, model: DpathExtractorModel, config: Config, **k
     def create_exponential_backoff_strategy(model: ExponentialBackoffStrategyModel, config: Config) -> ExponentialBackoffStrategy:
         return ExponentialBackoffStrategy(factor=model.factor, parameters=model.parameters, config=config)
 
-    def create_http_requester(self, model: HttpRequesterModel, config: Config, **kwargs) -> HttpRequester:
+    def create_http_requester(self, model: HttpRequesterModel, config: Config, *, name: str, **kwargs) -> HttpRequester:
```

Comment:
Does this still work if we get rid of `**kwargs` so `create_http_requester(self, model: HttpRequesterModel, config: Config, *, name: str)`? That would be more consistent to some of the other components that receive additional parameters.

I think the named args pattern works pretty well and we don't need `**kwargs` since it's not used anywhere